includes/definitions/adva.yaml

```diff
@@ -0,0 +1,12 @@
+os: adva
```

Comment:
Thx for going through. These are two OS. Hence, splitting it into two yaml. All the sysObjectID checks in the other files will then also disappear. The OS names are unfortunately not catchy. Called it adva_fsp150 and adva_fsp3kr7 now.